Assets/Scripts/Localization/LocalizationTable.cs

```diff
@@ -120,6 +174,82 @@ public static string[] GetLanguages()
             return localizationTable.Keys.ToArray();
         }
 
+        public static void LoadConfigFile(string pathToConfigFile)
+        {
+            localizationConfigurations = new Dictionary<string, LanguageData>();
+
+            if (File.Exists(pathToConfigFile) == false)
+            {
+                Debug.ULogErrorChannel("LocalizationTable", "No config file found at: " + pathToConfigFile);
```

Comment:
Thats assuming everything goes smoothly. If the config doesn't exist, then the localization code will attempt to redownload the latest version which will contain a clean config. However, if the user is offline then the download will obviously fail. In that case the lack of a config will cause a lot of weird behavior, including rtl languages not being rtl, not showing languages in the dropdown (it will fallback to codes), and other minor glitches in the localization system. I don't know if that is considered an error or warning, but regardless the results will be unexpected without a config.xml file.